model/categories.model.js

```diff
@@ -35,3 +35,32 @@ exports.selectReviewById = async (review_id) => {
     return Promise.reject();
   }
 };
+
+exports.updateReviewById = async (review_id, inc_votes) => {
+  try {
+    const {
```

Comment:
this is some super fancy code!

nested destructuring, with some array destructuring, ternary opertator all wrapped up in a `try`/`catch` `async`/`await` bow? 

*chefs kiss* 🤌

There is something to be said about readability here however - sometimes a good old `if` statement can make you and your teammates lives a little bit easier

Reply:
Thanks! Good point about readability, I split up some of the nested destructuring to help with that.